plugin/src/main/java/io/snyk/eclipse/plugin/runner/ProcessRunner.java

```diff
@@ -122,13 +127,18 @@ private void setupProcessBuilderBase(ProcessBuilder pb) {
 
     String authMethod = Preferences.getInstance().getPref(Preferences.AUTHENTICATION_METHOD);
     String token = Preferences.getInstance().getAuthToken();
-    if (token != null && authMethod.equals(Preferences.AUTH_METHOD_OAUTH)) {      
-      pb.environment().put(EnvironmentConstants.ENV_INTERNAL_SNYK_OAUTH_ENABLED, "1");
-      pb.environment().put(EnvironmentConstants.ENV_INTERNAL_OAUTH_TOKEN_STORAGE, token);
-      pb.environment().remove(EnvironmentConstants.ENV_SNYK_TOKEN);
+    if (token != null && authMethod.equals(Preferences.AUTH_METHOD_OAUTH)) {   
+      try {
+	      ObjectMapper objectMapper = new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
If I remember correctly, there's already code somewhere to parse the OAuth token - could we maybe extract that code to remove duplication?